tests/tools/test_bootstrap.py

```diff
@@ -1,9 +1,14 @@
 import pandas as pd
+import pytest
 
 from pharmpy.results import ModelfitResults, read_results
 from pharmpy.tools.bootstrap.results import calculate_results
 
 
+@pytest.mark.filterwarnings(
+    # FIXME Remove filter once altair stops relying on deprecated APIs
```

Comment:
Should we perhaps open an issue on Altair?

Reply:
Yes. I'll double check if it's an `altair` issue or `vega-lite` issue.

Reply:
Definitely `altair`. Will open an issue.

Reply:
See https://github.com/altair-viz/altair/issues/2698.